src/inputs/forms/json-schema-form/mock.ts

```diff
@@ -78,13 +78,25 @@ export const getDefaultSchema = () => ({
         },
         colors: {
             description: 'Favorite Colors',
-            title: 'Favorite Colors',
+            title: 'Favorite Colors (multi input)',
             type: 'array',
             prefixItems: [
                 { type: 'string' },
                 { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
             ],
-            default: ['gray', 'blue'],
+            default: ['gray'],
+            multiInputMode: true,
+        },
+        color: {
+            description: 'Favorite Color',
+            title: 'Favorite Color (single input)',
+            type: 'array',
+            prefixItems: [
+                { type: 'string' },
+                { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
+            ],
+            default: ['gray'],
+            multiInputMode: false,
```

Comment:
praise: haha 😂 I know very well that you must have been thinking about what kind of property to put in. Favorite Color is very goood 😄